.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen vs what actually happened.
+
+**Log**
+Please include the log information for the session where you encountered this error. Logs can be found in `freemocap_data/logs_info_and_settings/logs` or copy and pasted from the terminal. Include as much context as possible so we can see upstream errors.
```

Comment:
```suggestion
Please include the log information for the session where you encountered this error. Logs can be found in `[path_to_your_user_directory]/freemocap_data/logs_info_and_settings/logs` or copy and pasted from the terminal. Include as much context as possible so we can see upstream errors.
```

Add a hint for the full path to the `freemocap_data` folder. 

Some True N00bs might still be confused by that, but its better than nothing.